core/api/core.api

```diff
@@ -1,8 +1,13 @@
 public final class com/juul/kable/Advertisement {
-	public fun <init> (ILandroid/bluetooth/BluetoothDevice;)V
+	public fun <init> (Landroid/bluetooth/le/ScanResult;)V
 	public final fun getAddress ()Ljava/lang/String;
+	public final fun getManufacturerData ()Lcom/juul/kable/ManufacturerData;
 	public final fun getName ()Ljava/lang/String;
 	public final fun getRssi ()I
+	public final fun getTxPower ()Ljava/lang/Integer;
+	public final fun getUuids ()Ljava/util/List;
+	public final fun manufacturerData (I)[B
+	public final fun serviceData (Ljava/util/UUID;)[B
```

Comment:
For my own curiosity, what are these characters after the function definition? "(Ljava/util/UUID;)[B"

Something to do with transpilation, I assume, but what do they mean?

Reply:
Shorthand notation for Java bytecode.

[Here](http://homepage.divms.uiowa.edu/~slonnegr/wpj/JNI.pdf) is a short reference (for some of the symbols):
![Screen Shot 2021-01-04 at 10 47 54 AM](https://user-images.githubusercontent.com/98017/103568435-60af2600-4e7a-11eb-8a35-7fe03c8f4833.png)

Reply:
> Something to do with transpilation

Nothing to do w/ transpilation actually. Transpilation is for translating Kotlin to JavaScript.
There are Java (JVM) bytecode API dumps (not JavaScript at all).

Reply:
Oh, why is this needed? The compiler can't figure this out for itself?

Reply:
These API dumps (`.api`) are for our reference (not the compiler's). So we can observe how the API has changed. Simplifies the process of adhering to semver for our releases.

Reply:
Gotcha. Thanks!